core/templates/pages/community-dashboard-page/translation-language-select/translation-language-select.component.spec.ts

```diff
@@ -0,0 +1,192 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { TranslationLanguageSelectComponent } from
```

Comment:
Add a newline to separate local and angular imports

Reply:
Done